File: PrimeNumber.py
import math

start = int(input("Enter start of range: "))
end = int(input("Enter end of range: "))

print("Prime numbers in the range:")

for num in range(start, end + 1):
    if num < 2:
        continue
    prime = True
    for i in range(2, int(math.sqrt(num)) + 1):
        if num % i == 0:
            prime = False
            break
    if prime:
        print(num, end=" ")

File: PrimeNumber.cpp
#include <iostream>
#include <cmath>
using namespace std;

bool isPrime(int n) {
    if (n < 2) return false;
    for (int i = 2; i <= sqrt(n); i++) {
        if (n % i == 0)
            return false;
    }
    return true;
}

int main() {
    int start, end;

    cout << "Enter start of range: ";
    cin >> start;

    cout << "Enter end of range: ";
    cin >> end;

    cout << "Prime numbers in the range:\n";

    for (int num = start; num <= end; num++) {
        if (isPrime(num)) {
            cout << num << " ";
        }
    }

    return 0;
}
